spanner/utils.rb

```diff
@@ -14,12 +14,12 @@
 
 def run_command command, arguments, project_id
   sample_method = method command
-  parameters = { project_id: project_id }
+  parameters = {}
 
   sample_method.parameters.each do |paramater|
     next if paramater.last == :project_id
     parameters[paramater.last] = arguments.shift
   end
 
-  sample_method.call(**parameters)
+  sample_method.call(project_id: project_id, **parameters)
```

Comment:
@jiren Sorry that I might not make this clear in my previous comment. I thought we can do: 

```ruby
sample_method.call(project_id, *arguments)
```

So we can remove the code in Line 19-22. Is there any issue to do in this way?

Reply:
`sample_method.call(project_id, *arguments)` this will not work. We can not pass arguments directly because the method has named parameters so, it must be in the form of Hash.

i.e. `{  instance_id: 'intance', database_id: 'test-db' }`

Reply:
Okay, then it's fine. Thanks for the explanation.